### PR TITLE
Fixed distro symlinks

### PR DESCRIPTION
**Pull request by yuriw**
Pls review

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
Need to analyse this run on this wip http://pulpito.ovh.sepia.ceph.com:8081/teuthology-2016-04-09_16:54:04-teuthology-master---basic-openstack/

File: suites/teuthology/rgw/tasks/s3tests-fastcgi-civetweb.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# this runs s3tests against rgw, using mod_fastcgi
```
Comment (zmc) [inline suites/teuthology/rgw/tasks/s3tests-fastcgi-civetweb.yaml]: This change doesn't look like it belongs in this PR?

File: suites/teuthology/rgw/tasks/s3tests-civetweb.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# this runs s3tests against rgw, using civetweb
```
Comment (zmc) [inline suites/teuthology/rgw/tasks/s3tests-civetweb.yaml]: again, this doesn't seem to belong in this commit

Comment (yuriw) [inline suites/teuthology/rgw/tasks/s3tests-civetweb.yaml]: @zmc Let me move it to a separate PR

**Comment by zmc** [comment]
@yuriw - it's "symlinks" :)
